domain/src/main/java/org/jboss/set/mjolnir/archive/domain/RepositoryFork.java

```diff
@@ -2,15 +2,7 @@
 
 import org.hibernate.annotations.CreationTimestamp;
 
-import javax.persistence.Column;
-import javax.persistence.Entity;
-import javax.persistence.GeneratedValue;
-import javax.persistence.GenerationType;
-import javax.persistence.Id;
-import javax.persistence.JoinColumn;
-import javax.persistence.ManyToOne;
-import javax.persistence.SequenceGenerator;
-import javax.persistence.Table;
+import javax.persistence.*;
```

Comment:
If possible configure your IDE to avoid wildcard imports. In EAP related projects it's also not allowed so you will find it handy.

Reply:
Modified these changes